bootc/Containerfile.centos9

```diff
@@ -4,34 +4,57 @@ RUN rm -rf /etc/yum.repos.d/*.repo
 COPY output/yum.repos.d /etc/yum.repos.d
 
 ARG PACKAGES="\
-bind-utils \
-buildah \
-cephadm \
-chrony \
-cloud-init \
-crudini \
-crypto-policies-scripts \
-device-mapper-multipath \
-driverctl \
-grubby \
-iproute-tc \
-iptables-services \
-iscsi-initiator-utils \
-jq \
-lvm2 \
-nftables \
-numactl \
-openssh-server \
-openstack-selinux \
-openvswitch \
-os-net-config \
-podman \
-python3-libselinux \
-python3-pyyaml \
-rsync \
-tmpwatch \
-tuned-profiles-cpu-partitioning \
-sysstat"
+	bind-utils \
+	buildah \
+	cephadm \
+	chrony \
+	cloud-init \
+	crudini \
+	cronie \
+	crypto-policies-scripts \
+	device-mapper-multipath \
+	driverctl \
+	grubby \
+	grub2-tools \
+	iproute-tc \
+	iptables-services \
+	iscsi-initiator-utils \
+	jq \
+	libvirt \
+	libvirt-daemon-driver-qemu \
+	lvm2 \
+	NetworkManager-ovs \
+	nftables \
+	numactl \
+	openssh-server \
+	openstack-selinux \
+	openvswitch \
+	os-net-config \
+	podman \
+	python3-libselinux \
+	python3-pyroute2 \
+	python3-pyyaml \
+	rsync \
+	sysstat \
+	tmpwatch \
+	tuned-profiles-cpu-partitioning \
+	qemu-kvm"
+
 ARG ENABLE_UNITS="openvswitch"
 
-RUN dnf -y update && dnf -y install $PACKAGES && dnf clean all && systemctl enable $ENABLE_UNITS
+RUN sudo dnf install centos-release-nfv-openvswitch centos-release-openstack-dalmatian -y && dnf -y install $PACKAGES && dnf clean all && systemctl enable $ENABLE_UNITS
```

Comment:
where are you getting the rpms centos-release-nfv-openvswitch centos-release-openstack-dalmatian from? you must be customizing the repo-setup command for the build somehow?

those are available on the base [quay.io/centos-bootc/centos-bootc:stream9](http://quay.io/centos-bootc/centos-bootc:stream9) image, but since our first command is RUN rm -rf /etc/yum.repos.d/*.repo, the enabled repos get removed, and then those packages are no longer available